tests/pr_test.go

```diff
@@ -44,3 +45,33 @@ func TestRunUpgradeExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestPrivateInSchematics(t *testing.T) {
+	t.Parallel()
+
+	const testLocation = "examples/private"
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "sm-prv-crt-eng-private",
+		TarIncludePatterns: []string{
+			"*.tf",
+			testLocation + "/*.tf",
+		},
+		ResourceGroup:          resourceGroup,
+		TemplateFolder:         testLocation,
+		Tags:                   []string{"test-schematic"},
+		DeleteWorkspaceOnFail:  false,
+		WaitJobCompleteMinutes: 80,
```

Comment:
You should use `BestRegionYAMLPath: "../common-dev-assets/common-go-assets/cloudinfo-region-secmgr-prefs.yaml"` to ensure the test uses a region from this file - @toddgiguere this works for schematics test right?

Reply:
Yes, that will still work for the schematics selector.